src/Components/Footer/footer.css

```diff
@@ -1,12 +1,38 @@
+@import url('https://fonts.googleapis.com/css2?family=Mina&family=Montserrat:wght@800&display=swap');
+
 .footer-text p {
```

Comment:
are these changes related?

Reply:
actually, they are not. I accidentally carried over my changes from the footer update pr here. I fixed it.